packages/app/src/app/pages/Sandbox/Editor/Header/Buttons/PreferencesButton.tsx

```diff
@@ -1,14 +1,20 @@
-import React from 'react';
+import React, { FunctionComponent } from 'react';
 import SettingsIcon from 'react-icons/lib/md/settings';
-import { inject, hooksObserver } from 'app/componentConnectors';
+
+import { useOvermind } from 'app/overmind';
+
 import { Action } from './Action';
 
-export const PreferencesButton = inject('signals')(
-  hooksObserver(({ signals: { modalOpened } }) => (
+export const PreferencesButton: FunctionComponent = () => {
+  const {
+    actions: { modalOpened },
+  } = useOvermind();
+
+  return (
     <Action
-      onClick={() => modalOpened({ modal: 'preferences' })}
-      tooltip="Preferences"
       Icon={SettingsIcon}
+      onClick={() => modalOpened({ message: null, modal: 'preferences' })}
```

Comment:
Just like https://github.com/codesandbox/codesandbox-client/pull/2544#discussion_r328837587

Maybe make it optional? 🤔